zenpacklib.py

```diff
@@ -1935,15 +1975,19 @@ def __init__(
             :type impacted_by: list(str)
             :param monitoring_templates: TODO
             :type monitoring_templates: list(str)
-            :param filter_display: TODO
+            :param filter_display: Should this class show in any other filter dropdowns?
             :type filter_display: bool
+            :param filter_hide_from: Classes for which this class should not show in the filter dropdown.
+            :type filter_hide_from: list(class)
             :param dynamicview_views: TODO
             :type dynamicview_views: list(str)
             :param dynamicview_group: TODO
             :type dynamicview_group: str
             :param dynamicview_relations: TODO
             :type dynamicview_relations: dict
-            # TODO: should make this a spec class, not a plain dict.
+            :param extra_paths: TODO
```

Comment:
`ClassSpec.extra_paths` needs to be added to the documentation in docs/yaml-classes-and-relationships.rst.